### DISCUSSION TITLE
Where do I find the background images that come with GrandOrgue?

### DISCUSSION THREAD

**Discussion by DavidShaw1970** [GrandOrgue/grandorgue#893] [Help]
I was wondering where I can find the image files used in Grand Orgue and  Hauptwerk 1.0 ( Yes, I'm going WAY back!)? (Wood grain backgrounds, stop knobs, manual keys, pedal keys, labels and pistons)

**Comment by larspalo** [comment]
https://github.com/GrandOrgue/grandorgue/tree/master/src/images

**Reply by DavidShaw1970** [reply]
Thank you for your help!
